packages/eslint-plugin/src/configs/all.ts

```diff
@@ -55,7 +55,6 @@ export = {
     'no-dupe-class-members': 'off',
     '@typescript-eslint/no-dupe-class-members': 'error',
     '@typescript-eslint/no-duplicate-enum-values': 'error',
-    'no-duplicate-imports': 'off',
```

Comment:
Intentional?

Reply:
this happened when I ran `yarn generate-configs`
it was removed because the rule has been deprecated.
IDK why it was missed.